oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -826,6 +1023,7 @@ impl Runtime {
     pub fn node_start_instance<I>(
         &self,
         node_id: NodeId,
+        pretty_name: String,
```

Comment:
Maybe mention that this is only used for debugging purposes?

Reply:
Done.